core/templates/pages/exploration-editor-page/services/user-exploration-permissions.service.spec.ts

```diff
@@ -16,64 +16,66 @@
  * @fileoverview Unit tests for the UserExplorationPermissionsService.
  */
 
-// TODO(#7222): Remove the following block of unnnecessary imports once
-// the code corresponding to the spec is upgraded to Angular 8.
-import { UpgradedServices } from 'services/UpgradedServices';
-// ^^^ This block is to be removed.
+import { HttpErrorResponse } from '@angular/common/http';
+import { HttpClientTestingModule, HttpTestingController }
+  from '@angular/common/http/testing';
+import { TestBed, fakeAsync, flushMicrotasks } from '@angular/core/testing';
 
-require(
-  'pages/exploration-editor-page/services/' +
-  'user-exploration-permissions.service.ts');
-require('services/context.service.ts');
-require('services/contextual/url.service.ts');
+import { ContextService } from
+  'services/context.service';
+import { UserExplorationPermissionsService } from
+  'pages/exploration-editor-page/services/user-exploration-permissions.service';
+
+describe('User Exploration Permissions Service', () => {
+  let ueps: UserExplorationPermissionsService = null;
+  let contextService: ContextService = null;
+  let httpTestingController: HttpTestingController = null;
 
-describe('User Exploration Permissions Service', function() {
-  var ueps, ContextService, UrlService, $httpBackend;
   var sampleExplorationId = 'sample-exploration';
```

Comment:
Use `const` or `let` instead of `var` here and elsewhere

Reply:
Done.